lighthouse-viewer/app/src/lighthouse-report-viewer.js

```diff
@@ -160,7 +182,9 @@ class LighthouseViewerReport {
     return this.github.createGist(this.json).then(id => {
       ga('send', 'event', 'report', 'created');
 
+      this.disableShareButton();
```

Comment:
should this be before `this.github.createGist` so you can't click again while it's in progress?